### ISSUE TITLE
Color contrast issues in flex layout examples

### ISSUE THREAD

**Issue by mcoker** [patternfly/patternfly-next#1690]
We should address these so they don't fail a11y.

```
1 error found in: /layouts/Flex/examples-full/

1) Elements must have sufficient color contrast -- https://dequeuniversity.com/rules/axe/3.1/color-contrast -- impact: serious
      
-------html violation instances-------

<h2 class="example-title">
  Basic flex - <code>.pf-l-flex</code>
</h2>
<code>.pf-l-flex</code>
<h2 class="example-title">
  Flex with modified spacing - <code>.pf-l-flex.pf-m-space-items-xl</code>
</h2>
<code>.pf-l-flex.pf-m-space-items-xl</code>
<h2 class="example-title">
  Flex with modified spacing - <code>.pf-l-flex.pf-m-space-items-none</code>
</h2>
<code>.pf-l-flex.pf-m-space-items-none</code>
<h2 class="example-title">
  Individually spaced items.</h2>
<h2 class="example-title">
  Test 1</h2>
<h2 class="example-title">
  Test 2</h2>
<h2 class="example-title">
  Nested flex items</h2>
<h2 class="example-title">
  Mixed nesting flex items</h2>
<h2 class="example-title">
  Mixed nesting flex items</h2>
<h2 class="example-title">
  Inception grouping</h2>
<h2 class="example-title">
  Aligning right with align right. This solution will always align element right by setting margin-left: auto, including when wrapped.</h2>
<h2 class="example-title">
  Align right on single groups.</h2>
<h2 class="example-title">
  Align right on multiple groups.</h2>
<h2 class="example-title">
  Using <code>.pf-m-flex-grow</code> + <code>.pf-m-nowrap</code>.</h2>
<code>.pf-m-flex-grow</code>
<code>.pf-m-nowrap</code>
<h2 class="example-title">
  Using <code>.pf-m-flex-grow</code> + <code>.pf-m-nowrap</code> (responsive).</h2>
<code>.pf-m-flex-grow</code>
<code>.pf-m-nowrap</code>
<h2 class="example-title">
  justify-content flex end.</h2>
<h2 class="example-title">
  justify-content space-between.</h2>
<h2 class="example-title">
  justify-contenting flex start (use to reset flex-direction to default).</h2>
<h2 class="example-title">
  Column flex - <code>.pf-l-flex.pf-m-column</code>
</h2>
<code>.pf-l-flex.pf-m-column</code> 
```

**Comment by rachael-phillips** [comment]
@mattnolting is addressing this in his work currently.
